### PR TITLE
e2e test for details page

### PR DESCRIPTION
**Pull request by bojanaave**
Added new e2e test that run through details page

### CODE REVIEW

File: cypress/support/steps/main.steps.ts [hunk 1]
```diff
@@ -95,15 +106,22 @@ export const borrow = (
...
+          .should('not.be.disabled')
+          .click();
+        cy.get(`[data-cy=Modal] h2:contains("Borrow ${_shortName}")`).should('be.visible');
+      });
+    } else {
+      it('Open via details page', () => {
+        cy.contains('Your info').click();
+        cy.contains('Borrow').click();
+      });
+    }
```
Comment (MareskoY) [inline cypress/support/steps/main.steps.ts:124]: i guess that u don't need to change this one, cause u never use this functionality

File: cypress/support/steps/main.steps.ts [hunk 2]
```diff
@@ -49,13 +51,20 @@ export const supply = (
...
+          .find('button:contains("Supply")')
+          .click();
+        cy.get(`[data-cy=Modal] h2:contains("Supply ${_shortName}")`).should('be.visible');
+      });
+    } else {
+      it('Open via details page', () => {
+        cy.contains('Your info').click();
+        cy.contains('Supply').click();
+      });
+    }
```
Comment (MareskoY) [inline cypress/support/steps/main.steps.ts:67]: i guess that u don't need to change this one, cause u never use this functionality

File: cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      hasApproval: false,
+    },
+    borrow: [
+      {
+        asset: assets.aaveMarket.USDT,
+        amount: 50,
+        apyType: constants.borrowAPYType.variable,
+        hasApproval: true,
+      },
+    ],
```
Comment (MareskoY) [inline cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts:26]: don't need to us array here, u just have 1 borrow action, and don't need to use second one

File: cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  },
+};
+describe('Verify supply and borrow modals does appear', () => {
+  const skipTestState = skipState(false);
+  configEnvWithTenderlyMainnetFork({});
+  supply(testData.depositETH, skipTestState, true);
+  testData.testCases.borrow.forEach((borrowCase) => {
+    borrow(borrowCase, skipTestState, true);
+  });
```
Comment (MareskoY) [inline cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts:35]: this steps not requered, cause u don't verify values which u get

File: cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+  describe('Verifing collateral, e-mode and isolation strings on detail page USDT AAVE V2', () => {
```
Comment (MareskoY) [inline cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts:37]: wrong describe `e-mode and isolation strings on detail page USDT AAVE V2`
1. u don't check that
2. v2 version not contains option for e-mode and isolated mode

File: cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    it('Open details page for USDT', () => {
+      cy.get('[data-cy="menuMarkets"]').click();
+      cy.get('[data-cy="marketListItemListItem_USDT"]').click();
+      cy.contains('Asset cannot be used as collateral.');
+    });
+
+    it('step2: Switch to V3 market and check e-mode isolation and collateral strings USDT', () => {
```
Comment (MareskoY) [inline cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts:44]: is it step 2 where is step 1?

File: cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+      cy.contains('Go Back').click();
+      cy.get('[data-cy="marketSelector"]').click();
+      cy.contains('Arbitrum').click();
```
Comment (MareskoY) [inline cypress/integration/0-v2-markets/0-main-v2-market/details.page.aave-v2.spec.ts:47]: all of that could be changed by `configEnvWithTenderlyArbitrumFork({ v3: true });` also it save u from fluky result

**Review by MareskoY — COMMENTED** [review]
need to fix

**Review by MareskoY — APPROVED** [review]

**Review by defispartan — APPROVED** [review]

**Review by foodaka — APPROVED** [review]

**Pull request merged**